apps/form-system/web/src/components/ApplicationsLoading/ApplicationLoading.tsx

```
@@ -12,5 +12,6 @@ export const ApplicationLoading = () => (
     className={styles.root}
   >
     <LoadingDots large />
+    <>asdf</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)